pom.xml

```diff
@@ -244,6 +246,14 @@
                     <failOnError>false</failOnError>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>versions-maven-plugin</artifactId>
```

Comment:
we do not use maven versions plugin, but instead use renovate bot for updating dependencies.
https://docs.renovatebot.com/configuration-options/#allowedversions

Reply:
I will remove it.

Reply:
sorry, i've mixed the bot from the other projects.
https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#specifying-dependencies-and-versions-to-ignore
on github we use depedenabot and it also has the same configuration